docs/content/kafka-authorization.md

```diff
@@ -304,9 +390,7 @@ the `pii_consumer` credentials to simulate a service that is allowed to read
 PII data.
 
 ```bash
-docker run --rm --network opakafkatutorial_default \
-    openpolicyagent/demo-kafka:1.0 \
-    kafka-console-consumer --bootstrap-server kafka:9093 --topic credit-scores --from-beginning --consumer.config /etc/kafka/secrets/pii_consumer.ssl.config
+kafka-console-consumer.sh --bootstrap-server localhost:9093 --topic credit-scores --from-beginning --consumer.config cert/client/pii_consumer.properties
```

Comment:
Are these scripts contained in the kafka image, I think... can we make these commands run inside of Docker so that we're not relying on scripts being installed on the host?

Reply:
We'd need to mount the client certs into the Kafka broker container too, which conceptually felt.. odd. But I guess it makes sense to avoid having to download and run scripts from the Kafka distribution. I will see if we can mount them on invocation rather than the docker compose file, as that will at least provide some illusion of decoupling between client and server :)